soc/arm/st_stm32/stm32l4/Kconfig.series

```diff
@@ -13,6 +13,8 @@ config SOC_SERIES_STM32L4X
 	select SOC_FAMILY_STM32
 	select HAS_STM32CUBE
 	select CPU_HAS_ARM_MPU
+	select LPTIM_CLOCK_LSI
+	select LPTIM_CLOCK_LSE
```

Comment:
Can't be both...
Btw, is that required?

Reply:
removed